halo2_proofs/Cargo.toml

```diff
@@ -54,7 +54,7 @@ blake2b_simd = "1"
 maybe-rayon = {version = "0.1.0", default-features = false}
 
 # Developer tooling dependencies
-plotters = { version = "0.3.0", default-features = false, optional = true }
+plotters = { version = "0.3.0", optional = true }
```

Comment:
I turned on the default features to generate a test circuit layout, the PSE fork has the defaults on too.